src/Microsoft.DotNet.Build.Tasks/Targets/tests.targets

```diff
@@ -48,10 +49,20 @@
     <DebugEngines>{2E36F1D4-B23C-435D-AB41-18E608940038}</DebugEngines>
   </PropertyGroup>
 
+  <ItemGroup>
+    <TestTargetFrameworkPath Include="$(TestPath)%(TestTargetFramework.Folder)"/>
+    <TestsSuccessfulSemaphore Include="@(TestTargetFrameworkPath->'%(FullPath)\$(MSBuildProjectFile).tests.lastsucceeded')"/>
+    <XunitResultsFile Include="@(TestTargetFrameworkPath->'%(FullPath)\$(XunitResultsFileName)')"/>
+  </ItemGroup>
+
   <!-- On command line run unit tests on CoreCLR after the Test target -->
   <Target Name="RunTestsForProject"
           AfterTargets="CheckTestCategories"
+          Inputs="@(PackagesConfig);$(TestRuntimePackageConfig);@(ReferenceCopyLocalPaths);@(Content);@(IntermediateAssembly);@(_DebugSymbolsIntermediatePath)"
+          Outputs="@(TestsSuccessfulSemaphore);@(XunitResultsFile)"
           Condition="'$(TestDisabled)'!='true'">
+		  
+    <CallTarget Targets="CopyTestToTestDirectory"/>
```

Comment:
Why not just add a DependsOnTarget="CopyTestToTestDirectory"? Does that still force it to run even if the Inputs/Outputs are in-sync for this target?